test/features/router/misc.js

```diff
@@ -111,7 +111,7 @@ describe('router - misc', function() {
             throw new Error('Error should be thrown');
         } catch(e) {
             assert.deepEqual(e.constructor.name, 'HTTPError');
-            assert.deepEqual(e.body.detail, 'data.body.testParam is a required property');
+            assert.deepEqual(e.body.detail, "data.body should have required property 'testParam'");
```

Comment:
@d00rman this error message doesn't come from our code, it's produced by `ajv`, so we can't change that.